Gems/PhysX/Code/Editor/EditorJointConfiguration.cpp

```diff
@@ -330,6 +331,9 @@ namespace PhysX
                         "Select = Show setup display when entity is selected."
                         "Always = Always show setup display.")
                     ->Attribute(AZ::Edit::Attributes::ReadOnly, &EditorJointConfig::IsInComponentMode)
+                    ->DataElement(AZ::Edit::UIHandlers::Default, &EditorJointConfig::m_jointname 
+                        , "Joint name"
+                        , "Usually coming from URDF")
```

Comment:
The comment could be more descriptive. Given that "Joint name" is already quite generic, it might be hard for users to derive what this name is actually used for/by.

Reply:
I agree, maybe it is better not to erase it from the `editContext`. Finally it is going to be used only by developers.